src/components/PlatformsSupported/helpers/getPlayableOn.ts

```diff
@@ -0,0 +1,41 @@
+import { SupportedPlatform } from '@valist/sdk'
+
+import { Platform } from '../../PlatformIcon/types'
+import { getPlatformsBuiltFor } from './getBuiltFor'
+import { getPlatformsFromSupportedPlatforms } from './getPlatforms'
+
+function getPlayableOn(platform: SupportedPlatform): Platform[] {
```

Comment:
this method is so repeated that maybe we should move it to the utils package, because we have a similar one on the client as well.